src/main/scala/com/gu/datalakealerts/Features.scala

```diff
@@ -82,7 +82,7 @@ object Features {
             |and ab.name like '%epic%'
             |and ab.completed = True
             |group by 1
-          """.stripMargin, 285000)
+          """.stripMargin, 42680)
```

Comment:
Olgil threshold set as half of yesterday's total impressions, as discussed with @jordanterry